filament/src/ToneMapper.cpp

```diff
@@ -238,7 +238,7 @@ DEFAULT_CONSTRUCTORS(DisplayRangeToneMapper)
 
 float3 DisplayRangeToneMapper::operator()(math::float3 c) const noexcept {
     // 16 debug colors + 1 duplicated at the end for easy indexing
-    constexpr float3 debugColors[17] = {
+    float3 debugColors[17] = {
```

Comment:
I tried this code with gcc 12.1 and it works.

In any case, try `static constexpr` instead. If it doesn't work, use `static const`.